README.md

```diff
@@ -33,106 +39,102 @@ input:
+custom configuration parameters, you can create a `config.yaml` with your desired
+parameters. For example:
 
 ```yaml
 deployers:
   image: 
-    deployer_name: docker|podman|kubernetes
+    deployer_name: podman
+    deployment:
+      host:
+        NetworkMode: host
+      imagePullPolicy: IfNotPresent
   python:
     deployer_name: python
-  # More deployer options
 log:
-  level: debug|info|warning|error
+  level: debug
+logged_outputs:
+  error:
+    level: debug
 ```
 
-You can load this config by passing the `-config` flag to Arcaflow.
+You can load this config by passing the `--config` flag to Arcaflow.
 
-### Supported Workflow Schema Versions
+```bash
+arcaflow --input input.yaml --config config.yaml
+```
 
-- v0.2.0
+The default workflow file name is `workflow.yaml`, but you can override this with the
+`--workflow` input parameter.
 
-## Deployer options
+Arcaflow also accepts a `--context` parameter that defines the base directory for all
+input files. All relative file paths are from the context directory, and absolute paths
+are also supported. The default context is the current working directory (`.`).
 
-Currently, the two deployer types supported are Docker and Kubernetes.
+### A few command examples...
 
-### The Docker deployer
+Use the built-in configuration and run the `workflow.yaml` file from the `/my-workflow`
+context directory with no input:
+```bash
+arcaflow --context /my-workflow
+```
 
-This deployer uses the Docker socket to launch containers. It has the following config structure:
+Use a custom `my-config.yaml` configuration file and run the `my-workflow.yaml` workflow
+using the `my-input.yaml` input file from the current directory:
+```bash
+arcaflow --config my-config.yaml --workflow my-workflow.yaml --input my-input.yaml
+```
 
-```yaml
-image:
-  deployer_name: docker
-  connection:
-    host: # Docker connection string
-    cacert: # CA certificate for engine connection in PEM format
-    cert: # Client cert in PEM format
-    key: # Client key in PEM format
-  deployment:
-    container: # Container options, see https://docs.docker.com/engine/api/v1.41/#tag/Container/operation/ContainerCreate
-    host: # Host options, see https://docs.docker.com/engine/api/v1.41/#tag/Container/operation/ContainerCreate
-    network: # Network options, see https://docs.docker.com/engine/api/v1.41/#tag/Container/operation/ContainerCreate
-    platform: # Platform options, see https://docs.docker.com/engine/api/v1.41/#tag/Container/operation/ContainerCreate
-
-    # Pull policy, similar to Kubernetes
-    imagePullPolicy: Always|IfNotPresent|Never
-  timeouts:
-    http: 15s
+Use a custom `config.yaml` configuration file and the default `workflow.yaml` file from
+the `/my-workflow` context directory, and an `input.yaml` file from the current working
+directory:
+```bash
+arcaflow --context /my-workflow --config config.yaml --input ./input.yaml
```

Comment:
Good catch. In my imagination it worked (and I kind of wish it actually did).

Reply:
Well, it *could* work, with the default "context" value, or if we added `--context-input` and `--context-workflow` and `--context-config`, although that would probably be considered vaguely silly ...

But, yeah, if we have a `--context` to set the default path resolution base directory, we should use it consistently, or everyone will be confused instead of only some people. (Or maybe that's a goal!)

Reply:
> I kind of wish it actually did

@jaredoconnell was musing that we should provide special handling for a relative path starting with `./`.  So, yeah, I sympathize, but ultimately, on Unix, `./` is a _relative_ path, and we probably shouldn't mess with that notion.  (And, as I told @jaredoconnell, it's not that hard to type `$PWD` instead of `.`.  😉)